js/cart/cart-menu.js

```diff
@@ -0,0 +1,30 @@
+import { $ } from '../base/base.js';
+import { PRODUCT_STORAGE_KEY } from '../base/constant.js'
+import miniCart from '../component/cart-mini.js'
+
+const megamenuCart = $(".mega-menu--cart")
+
+const cartMenu = {
+    products: JSON.parse(localStorage.getItem(PRODUCT_STORAGE_KEY)) || [],
+
+    loadCart: function() {
+        let totalPrice = 0;
+        const cart = this.products.map((product, index) => {
+            totalPrice = totalPrice + product.sell
+            return miniCart(product);
+        })
+        const htmls = `${cart}
+            <div class="mega-menu__price">
+                <span class="mega-menu__total">Tổng số</span>
+                <span class="card__price">${totalPrice}đ</span></div>
+            <div class="button button-dark">Giỏ Hàng</div>
+        `
+        megamenuCart.innerHTML = htmls;
```

Comment:
Phân biệt giúp Việt innerHTML, innerText và textContent với nhé

Reply:
innerHTML : sẽ in ra nội dung text và tất cả những thẻ con và thuộc tính của nó
innerText: : sẽ in ra nội dung text và nội dung của các thẻ con bên trong (không có thẻ con chỉ có nội dung thẻ con)
textContent: chỉ in ra nội dung thẻ cha